code/__HELPERS/mobs.dm

```diff
@@ -54,8 +54,8 @@ proc/random_name(gender, speciesName = "Human")
 	if(S)
 		return S.makeName(gender)
 	else
-		var/datum/species/human/H
-		return H.makeName(gender)
+		return call(/datum/species/proc/makeName)(gender)

```

Comment:
Ideally the all_species list should exist well before this random_name proc is ever being called, it's weird that it isn't but on local servers this is the case 100% of the time